src/routes/history/+page.svelte

```diff
@@ -30,6 +30,10 @@
     </div>
 
     <div class="overflow-y-auto p-2.5">
+        {#if data.history.length === 0}
+            <div class="history-message-none">{$t['History_None']}</div>
+            <div class="history-message-none-info">{$t['History_None_Info']}</div>
+        {/if}
```

Comment:
Please put the iteration of the data.history items in an else branch of the if condition